### PR TITLE
migrate_vm: Update hugepage number

### PR DESCRIPTION
**Pull request by cliping**
The value of 'remote_target_hugepages' need less than vm mermory.
If this value is too large, migration will successful.

Signed-off-by: lcheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 virsh.migrate_vm.negative_testing.live_migration.with_hugepages.target_hugemem_less_than_vm_mem.without_postcopy --vt-connect-uri qemu:///system
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : e9b548feae7138ab16d5d94b283a765085985ffe
JOB LOG    : /root/avocado/job-results/job-2021-07-29T11.47-e9b548f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_vm.negative_testing.live_migration.with_hugepages.target_hugemem_less_than_vm_mem.without_postcopy: PASS (204.50 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 206.23 s

File: libvirt/tests/cfg/migration/migrate_vm.cfg [hunk 1]
```diff
@@ -642,7 +642,7 @@
                                     setup_hugepages = "yes"
                                     mb_enable = "yes"
                                     config_remote_hugepages = "yes"
-                                    remote_target_hugepages = -100
+                                    remote_target_hugepages = -700
```
Comment (Yingshun) [inline libvirt/tests/cfg/migration/migrate_vm.cfg:645]: As discussed privately, target_hugepages should be set to less than min(src_hugepage_name, vm_mem_size/2048). So we should fix this problem in code.

Comment (cliping) [reply libvirt/tests/cfg/migration/migrate_vm.cfg]: Updated.

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 virsh.migrate_vm.negative_testing.live_migration.with_hugepages.target_hugemem_less_than_vm_mem.without_postcopy --vt-connect-uri qemu:///system
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 0dffb7fc177e0a7823247ef271cd9fdb75e75070
JOB LOG    : /root/avocado/job-results/job-2021-07-30T03.46-0dffb7f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_vm.negative_testing.live_migration.with_hugepages.target_hugemem_less_than_vm_mem.without_postcopy: PASS (199.57 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 200.87 s

**Review by Yingshun — APPROVED** [review]
LGTM

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**